controllers/Backend.php

```diff
@@ -70,26 +95,42 @@ public function postback()
         $model_article = new Article();
         $model_user = new User();
         
+            if($model_user->admin()!=1){
+        header("Location:/dashboard");
+    }
+
+    $post = $model_article->get_post();
+    if($post == false){
+        header("Location:/page-erreur-administrateur");
+    }
+    
         $page="backend/postback";
-        $topbar="backend/topbar";
+        $topbar="topbar_backend";
         require "views/layout.php";
     }
 
     public function login()
     {
         $model_user = new User();
         
+            if(isset($_SESSION['admin'])){
+        header("Location:/dashboard");
+    }
+    
         $page="backend/login";
-        $topbar="backend/topbar";
+        $topbar="topbar_backend";
         require "views/layout.php";
     }
     
     public function new()
     {
         $model_user = new User();
         
+        if(isset($_SESSION['admin'])){
+        header("Location:/dashboard");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)